ui/components/MeshModelRegistry/MeshModelDetails.js

```diff
@@ -404,7 +396,9 @@ const Description = ({ description }) => (
 const TitleWithImg = ({ displayName, iconSrc }) => (
   <div style={{ display: 'flex', alignItems: 'center', flexBasis: '60%' }}>
     {iconSrc && <img src={iconSrc} height="55px" width="55px" style={{ marginRight: '0.6rem' }} />}
-    <Title title={displayName} />
+    <UsesSistent>
```

Comment:
don't use the usesSistent in small codes try to wrap it with the parent

Reply:
Done @amitamrutiya only the parent is now wrapped with usesSistent.